Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM ubuntu:latest
+
+RUN apt update
+
+ENV DEBIAN_FRONTEND=noninteractive
+
+RUN apt-get -y install tzdata
+
+RUN apt install -y build-essential cmake ninja-build libpcap-dev libglib2.0-dev libgcrypt20-dev libc-ares-dev bison flex liblz4-dev libsmi2-dev libgnutls28-dev libminizip-dev libbrotli-dev libsnappy-dev libzstd-dev libnghttp2-dev lua5.1 luajit libspandsp-dev libxml2-dev liblua5.1-dev libluajit-5.1-dev libkrb5-dev python3-pip python3-setuptools
```

Comment:
See this link to learn how to optimize `apt` usage for smaller image size
https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#apt-get